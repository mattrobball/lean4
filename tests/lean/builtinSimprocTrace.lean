def f (_ : Nat) := 10

set_option tactic.simp.trace true
example : f x = (if true then 8 + 2 else 0) := by
 simp
 rw [f]
