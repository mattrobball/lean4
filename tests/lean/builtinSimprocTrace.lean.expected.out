Try this: simp only [â†“reduceIte, Nat.reduceAdd]
